app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim

```diff
@@ -1,75 +1,76 @@
-                                    icon_name: 'exclamation-triangle',
-                                    notice: true
+        - unread_alerts = current_account.buyer_alerts.by_service(service).unread
+        - if can?(:manage, :monitoring) && unread_alerts.any?
+          li.DashboardNavigation-list-item
+            = dashboard_collection_link 'Alert',
+                                        unread_alerts,
+                                        admin_service_alerts_path(service),
+                                        icon_name: 'exclamation-triangle',
+                                        notice: true
 
-    // Subscriptions & Service Plans
-    - if show_subscriptions_on_dashboard?(service)
-      li.DashboardNavigation-list-item
-        // Subscriptions (would be nice to have API specific links)
-        => dashboard_collection_link 'Subscription',
-                                     service.service_contracts,
-                                     admin_buyers_service_contracts_path(search: {service_id: service.id}),
-                                     icon_name: 'paperclip'
+        // Subscriptions & Service Plans
+        - if show_subscriptions_on_dashboard?(service)
+          li.DashboardNavigation-list-item
+            // Subscriptions (would be nice to have API specific links)
+            => dashboard_collection_link 'Subscription',
+                                         service.service_contracts,
+                                         admin_buyers_service_contracts_path(search: {service_id: service.id}),
+                                         icon_name: 'paperclip'
 
-    // Active docs
-    - if can?(:manage, :plans)
-      li.DashboardNavigation-list-item
-       = dashboard_collection_link 'ActiveDoc',
-                                    service.api_docs_services,
-                                    admin_service_api_docs_path(service),
-                                    icon_name: 'file-text'
-    // Integration
-    - if can?(:manage, :plans)
-      li.DashboardNavigation-list-item
-        - if service.has_traffic? || (apiap? && service.proxy.proxy_configs.sandbox.any?)
-          - if has_out_of_date_configuration?(service)
-            = dashboard_navigation_link 'Integration',
-                                        path_to_service(service),
-                                        icon_name: 'wrench',
-                                        notice: true,
-                                        title: 'The Integration is out of date, promote!',
-                                        icon_append_name: 'exclamation-triangle'
-          - else
-            = dashboard_navigation_link 'Integration',
-                                        path_to_service(service),
-                                        icon_name: 'wrench'
+        // Active docs
+        - if can?(:manage, :plans)
+          li.DashboardNavigation-list-item
+           = dashboard_collection_link 'ActiveDoc',
```

Comment:
I noticed this line has a wrong indentation too and it causes the anchor to be rendered outside the `li`.
```suggestion
            = dashboard_collection_link 'ActiveDoc',
```